packages/components/src/components/buttons/Button/Button.stories.tsx

```diff
@@ -76,12 +76,10 @@ export const Button: StoryObj<ButtonProps> = {
             },
         },
         icon: {
-            options: {
-                'No icon': null,
-                ...variables.ICONS.reduce((acc, icon) => ({ ...acc, [icon]: icon }), {}),
-            },
+            options: variables.ICONS,
```

Comment:
can we set no icon?

I think it should be `[null, ...variables.ICONS]` like in `TextButton`